src/index.js

```diff
@@ -73,12 +78,88 @@ export function display() {
               <li class="item">
               ${item.description} 
               </li>
-              <i class="fas fa-ellipsis-v"></i>
+              <i id=${item.id}  class="fas fa-trash-alt"></i>
           </form>
           `;
       }
     });
     localStorage.setItem('todoLists', JSON.stringify(todoLists));
   }
 }
-display();
\ No newline at end of file
+display();
+
+const input = document.getElementById('input');
+const enter = document.getElementById('enter-key');
+function indexLoop() {
+  if (todoLists.length > 0) {
+    let i = 0;
+    while (i < todoLists.length) {
+      todoLists[i].index = i + 1;
+      i += 1;
+    }
+  }
+  return todoLists;
+}
+
+function addLocalStorage() {
+  indexLoop();
+  const localStore = JSON.stringify(todoLists);
+  localStorage.setItem('todoLists', localStore);
+  display();
+  window.location.reload(false);
+}
+
+enter.addEventListener('click', () => {
+  todoLists.push({ description: input.value, completed: false, index: todoLists.length });
+  display();
+  addLocalStorage();
+});
+if (localStorage.getItem('todoLists') !== null) {
+  todoLists = JSON.parse(localStorage.getItem('todoLists'));
+  display();
+}
+
+function remove(id) {
+  todoLists.splice(id, 1);
+  display();
+  addLocalStorage();
+}
+const btn = document.getElementsByClassName('fa-trash-alt');
+Array.from(btn).forEach((item, i) => {
+  item.addEventListener('click', () => {
+    remove(i);
+  });
+});
+
+function clear() {
+  todoLists = todoLists.filter((item) => item.completed === false);
+  display();
+  addLocalStorage();
+}
+
+const clean = document.getElementsByClassName('clear-completed');
+Array.from(clean).forEach((btn) => {
+  btn.addEventListener('click', () => {
+    clear();
+  });
+});
+// edit
+const area = document.getElementsByClassName('textarea');
+function edittodo(i) {
+  area[i].style.display = 'inline';
+  const line = document.querySelectorAll('.item');
+  line[i].style.display = 'none';
+  area[i].addEventListener('change', () => {
+    area[i].textContent = area[i].value;
+    todoLists[i].description = area[i].textContent;
+    addLocalStorage();
+  });
+}
```

Comment:
- [ ] OPTIONAL: When I edit a to-do item, It takes the text and hidden characters as textContent. So when I edit and hit 'enter', it takes the ↵ character along with the todo description text. Kindly fix this issue.  :pray: 

![Screenshot from 2021-11-06 14-59-56](https://user-images.githubusercontent.com/64894927/140608960-93ee0f1d-b90c-4596-9503-2deb9b702d2f.png)